apps/client/ui/Slider/Slider.tsx

```diff
@@ -1,82 +1,86 @@
-import { useState, useEffect, useRef } from 'react'
-import type { FC } from 'react'
+import React, { FC, ComponentPropsWithoutRef, ReactNode } from 'react';
 import 'rc-slider/assets/index.css';
 import Slider from 'rc-slider';
+import clsx from 'clsx'
 
-type CustomSliderProps = {
-  min?: number
-  max?: number
-  step?: number
-  defaultValue?: number
-  className?: string
+type CustomSliderProps = ComponentPropsWithoutRef<'input'> & {
+  range?: boolean;
+  min?: number;
+  max?: number;
+  step?: number;
+  defaultValue?: number | number[];
+  className?: string;
+  handleRender?: ReactNode;
 };
 
+const commonTrackStyle = [{ backgroundColor: 'var(--primary-color)', height: '4px' }];
```

Comment:
It's a place you can simplify - you can style these slider elements with classes instead of `style` props 👌 
You need to:
- create `Slider.css` file in the same directory
- find correct classes in the Slider (in the browsers dev-tools) and style them in the CSS file with `@apply` word and Tailwind classes instead of classic CSS rules.
- import CSS file into component file - `import './Slider.css'`

Then you should be able ot remove all CSS variables from `global.css` and keep your theme simple and consist 🤌

Reply:
you can add box-shadow value to `tailwind.config.js` like here - https://tailwindcss.com/docs/box-shadow

Reply:
I wanted to replace the shadow class in the tailwind.config.js (theme.boxShadow) but got a syntax error: 'The `shadow` class does not exist. If `shadow` is a custom class, make sure it is defined within a `@layer` directive.'